.eslintrc.js

```diff
@@ -10,7 +10,9 @@ module.exports = {
     jest: true,
   },
 
-  rules: {},
+  rules: {
+    'import/dynamic-import-chunkname': 'off',
```

Comment:
Since this PR will already change a lot of our chunks, I think it's a good time to include this change